tests/standalone_tests/test_celery_app.py

```diff
@@ -148,10 +172,9 @@ def test_celery_app_queue_task_with_rabbitmq_down(
     ), "The celery apps should NOT be the same, the rabbitmq is down causing a reset."
```

Comment:
you should probably check that `CeleryAppFactory().get_celery_app(RABBITMQ_LOCALNODETMP_ADDR)._celery_app` holds a `Celery` object also thought..

Reply:
fixed.

Reply:
Since you added that check as well, it would make sense to have it everywhere that you assert that the `Celery` has changed.
in `test_celery_app_is_different_after_get_result_when_rabbitmq_restarted` and
`test_celery_app_didnt_change_too_many_times_after_parallel_get_task_result_when_rabbitmq_restarted`

Reply:
In these tests I am checking afterwards that an execute task will work properly, so I didn't think there was a point to assert it there as well, since we don't want many asserts as well. Anyway, I will add it for consistency.